common/src/main/resources/docker/docker.bom

```diff
@@ -263,6 +270,7 @@ brooklyn.catalog:
         - type: org.apache.brooklyn.enricher.stock.Transformer
           brooklyn.config:
             uniqueTag: docker-bind-url-generator
+            enricher.suppressDuplicates: false
             enricher.triggerSensors:
               - $brooklyn:sensor("docker.port")
             enricher.targetSensor: $brooklyn:sensor("docker.bind.url")
```

Comment:
Andrew's #329 also moved the env `DOCKER_ENDPOINT_PUBLIC: $brooklyn:attributeWhenReady("docker.endpoint.public")` from docker.bom into swarm.bom. I see you haven't done that here. Agree it's unrelated to the purpose of this PR: the version upgrade + adding prometheus.